Dockerfile

```diff
@@ -16,20 +17,15 @@ COPY setup-r-envt.R setup-r-envt.R
 
 RUN Rscript setup-r-envt.R
 
-RUN conda config --add channels conda-forge
-
-RUN conda config --set channel_priority strict
-
 COPY environment.yml environment.yml
-
-RUN conda update -n base -c defaults conda
-
-RUN conda env update -n root -f environment.yml
+RUN conda config --add channels conda-forge \
```

Comment:
We should try to put as many docker commands as we can into a single one to avoid bloating the docker image. But this change is not related to the problem we are trying to solve with the env.

Reply:
wow - i didn't know that streamlines the build!! thank you @ocefpaf